HitDelay.py

```
@@ -1,429 +1,439 @@
+		os.system('start notepad ./musync_data/Acc-Sync.json')
+		# os.system(f'start explorer {os.getcwd()}')
+		import AvgAcc_SynxAnalyze
+		AvgAcc_SynxAnalyze.Analyze()
+
+	def ShowHistoryInfo(self,event):
+		e = event.widget									# 取得事件控件
+		itemID = e.identify("item",event.x,event.y)			# 取得双击项目id
+		# state = e.item(itemID,"text")						# 取得text参数
+		historyItem = e.item(itemID,"values")				# 取得values参数
+		if not self.history == []:
+			isChange = False
```

Ruff (0.14.8)

Local variable `isChange` is assigned to but never used

Remove assignment to unused variable `isChange`

(F841)